modules/tinymce/src/plugins/fullscreen/test/ts/browser/FullScreenPluginTest.ts

```diff
@@ -37,7 +37,8 @@ describe('browser.tinymce.plugins.fullscreen.FullScreenPluginTest', () => {
 
   const assertApiAndLastEvent = (editor: Editor, state: boolean) => {
     assert.equal(editor.plugins.fullscreen.isFullscreen(), state, 'Editor isFullscreen state');
-    assert.equal(lastEventArgs.get().state, state, 'FullscreenStateChanged event state');
+    assert.deepEqual(firedEvents, [ 'fullscreenstatechanged', state, 'resizeeditor', state ], 'FullscreenStateChanged event state');
+    firedEvents = [];
```

Comment:
This is pretty confusing to have here... as this is only meant to be an assertion but it's changing test state. Please change this so that this function doesn't alter state.